### PR TITLE
Updating Probability figures in Dan's chapter

### PR DESCRIPTION
**Pull request by amandakube**

### CODE REVIEW

File: textbook/11/1/Rules_Definitions.ipynb [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "\n",
     "**Complement of an event**: The complement of an event $A$ is the event that $A$ does not occur, denoted by $A^C$. For the event $A$ defined above, $A^C$ is the event that more than half of the babies are boys, or the set of integers from 50 to 100.\n",
     "\n",
-    "![](./img/venn_comp.png)\n",
+    "<img src=\"./img/complement.png\" width=\"400\"/> \n",
```
Comment (jesteria) [inline textbook/11/1/Rules_Definitions.ipynb:21]: Any reason to use `` and to set a width?

* As far as I can tell, one of the advantages to using the Markdown-native syntax is that the textbook builder detects what you're doing and ensures the image is included. With this version, none of the images are included and these tags are broken. For that reason, I think we don't really want to use `` tags at all.
* The width is probably fine, but generally it's nice to allow the browser to figure out the best sizing, based on the image and layout. (And if we do need to control the sizing, there might be a more dynamic way.)

I would just tweak these to work with the textbook – the Markdown syntax is both textbook-compatible and less verbose – but, let me know if there's a reason behind the particular form you used.

image

**Review by jesteria — CHANGES_REQUESTED** [review]
Cool. Question about the use of the `` tag (below).

**Comment by amandakube** [comment]
I changed it so that I could set a width because when I previewed it, the image was much too large. If you know a different way to make it smaller, that is ok. I just couldn’t make it work in markdown.

---
Amanda Kube, MS, PhD
Preceptor in Data Science
Data Science Institute
The University of Chicago
***@***.******@***.***>


From: Jesse London ***@***.***>
Date: Monday, November 7, 2022 at 11:29 AM
To: chicago-cdac/textbook-datascience-1 ***@***.***>
Cc: Amanda Kube ***@***.***>, Author ***@***.***>
Subject: Re: [chicago-cdac/textbook-datascience-1] Updating Probability figures in Dan's chapter (PR #29)

@jesteria requested changes on this pull request.

Cool. Question about the use of the  tag (below).

________________________________

In textbook/11/1/Rules_Definitions.ipynb<https://urldefense.com/v3/__https:/github.com/chicago-cdac/textbook-datascience-1/pull/29*discussion_r1015708014__;Iw!!BpyFHLRN4TMTrA!8eKfeU57HAXDfes4B5ZFzyVwuD9CZx_HcfwiKRGIxMr9GLR8Iy-B6D2JWuXkAFvQAP1eZB0ozxQ7L5_ewqOxD3g0Ng$>:

> @@ -18,7 +18,7 @@

     "\n",

     "**Complement of an event**: The complement of an event $A$ is the event that $A$ does not occur, denoted by $A^C$. For the event $A$ defined above, $A^C$ is the event that more than half of the babies are boys, or the set of integers from 50 to 100.\n",

     "\n",

-    "\n",

+    " \n",

Any reason to use  and to set a width?

  *   As far as I can tell, one of the advantages to using the Markdown-native syntax is that the textbook builder detects what you're doing and ensures the image is included. With this version, none of the images are included and these tags are broken. For that reason, I think we don't really want to use  tags at all.
  *   The width is probably fine, but generally it's nice to allow the browser to figure out the best sizing, based on the image and layout. (And if we do need to control the sizing, there might be a more dynamic way.)

I would just tweak these to work with the textbook – the Markdown syntax is both textbook-compatible and less verbose – but, let me know if there's a reason behind the particular form you used.

[image]<https://urldefense.com/v3/__https:/user-images.githubusercontent.com/530998/200374685-d70ba13b-80bc-4aa7-bfa4-f919f39ade06.png__;!!BpyFHLRN4TMTrA!8eKfeU57HAXDfes4B5ZFzyVwuD9CZx_HcfwiKRGIxMr9GLR8Iy-B6D2JWuXkAFvQAP1eZB0ozxQ7L5_ewqPnkDmazg$>

**Comment by jesteria** [comment]
OK cool. For reference, the textbook-standard syntax for setting such extended attributes is using either the `image` or the `figure` directive [[ref]](https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#syntax-images).

(HTML is otherwise not parsed by the textbook builder, rather passed directly through as text. This is mainly an issue for images, which require extra processing.)

That said, there's an extension to catch `` tags and convert them … so why not? I enabled the extension.

As for the static width, sure, 400px seems perfectly reasonable for this layout. (The layout appears to correctly handle the images regardless of the size of the window/viewport.)

If you think it helpful, I'll just see to it that the images are horizontally centered.

**Review by jesteria — APPROVED** [review]

**Pull request merged**